### PR TITLE
[Queues] Add notes on consumer concurrency and retries

### PR DESCRIPTION
**Pull request by kewbish**
This PR adds several warning notes about retried messages causing consumers to autoscale down. Several users on Discord were confused about this since this behaviour isn't explicitly stated in the docs. 

(Inclusive language check failure is not part of my PR, but I can address it if desired!)

TODO:
- [x] Once #10156 is merged, rebase and add the same warning to the Wrangler bindings page

### CODE REVIEW

File: content/queues/learning/batching-retries.md [hunk 1]
```diff
@@ -103,6 +103,12 @@ Each retry counts as an additional read operation per [Queues pricing](/queues/p
 
 When a single message within a batch fails to be delivered, the entire batch is retried, unless you have [explicitly acknowledged](#explicit-acknowledgement) a message (or messages) within that batch. For example, if a batch of 10 messages is delivered, but the 8th message fails to be delivered, all 10 messages will be retried and thus redelivered to your consumer in full.
 
+{{<Aside type="warning" header="Retried messages and consumer concurrency">}}
+
+Retrying messages with `.retry()` or calling `.retryAll()` on a batch will cause the consumer to autoscale down if consumer concurrency is enabled. Refer to [Consumer concurrency](/queues/learning/consumer-concurrency/) to learn more.
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/batching-retries.md:108]:
```suggestion
Retrying messages with `retry()` or calling `retryAll()` on a batch will cause the consumer to autoscale down if consumer concurrency is enabled. Refer to [Consumer concurrency](/queues/learning/consumer-concurrency/) to learn more. 
```

File: content/queues/learning/consumer-concurrency.md [hunk 2]
```diff
@@ -26,6 +26,12 @@ The number of consumers concurrently invoked for a queue will autoscale based on
 
 Where possible, Queues will optimize for keeping your backlog from growing exponentialy, in order to minimize scenarios where the backlog of messages in a queue grows to the point that they would reach the [message retention limit](/queues/platform/limits/) before being processed.
 
+{{<Aside type="warning" header="Consumer concurrency and retried messages">}}
+
+Retrying messages with `.retry()` or calling `.retryAll()` on a batch will count as a failed invocation and cause the consumer to autoscale down. If your consumer concurrency remains at 1 but your consumer's `max_concurrency` is something higher, it is usually due to messages being retried, preventing your consumer from scaling up.
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/consumer-concurrency.md:31]:
```suggestion
Retrying messages with `retry()` or calling `retryAll()` on a batch will count as a failed invocation and cause the consumer to autoscale down. If your consumer concurrency remains at 1 but your consumer's `max_concurrency` is higher, it is usually due to messages being retried, preventing your consumer from scaling up.
```

File: content/queues/platform/configuration.md [hunk 3]
```diff
@@ -84,5 +84,6 @@ Refer to [Limits](/queues/platform/limits) to review the maximum values for each
 - {{<code>}}max_concurrency{{</code>}} {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
 
   - The maximum number of concurrent consumers allowed to run at once. Leaving this unset will mean that the number of invocations will scale to the [currently supported maximum](/queues/platform/limits/).
+  - Refer to [Consumer concurrency](/queues/learning/consumer-concurrency/) for more information on how consumers autoscale, particularly when messages are retried.
```
Comment (Maddy-Cloudflare) [inline content/queues/platform/configuration.md:87]:
```suggestion
  - Refer to [Consumer concurrency](/queues/learning/consumer-concurrency/) for more information on how consumers autoscale when messages are retried.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Comment by kewbish** [comment]
(Going to merge since the line I just added was already reviewed by PCX and was just copy-pasted from the Queues binding documentation on the Queues page to the Queues binding documentation on the Wrangler configuration page.)

**Pull request merged**